actnow/profiles/models.py

```diff
@@ -11,13 +11,13 @@
 class UserProfile(TimestampedModelMixin):
     user = models.OneToOneField(get_user_model(), on_delete=models.CASCADE)
     first_name = models.CharField(_("first name"), max_length=255)
-    last_name = models.CharField(_("last name"), max_length=255, null=True, blank=True)
-    bio = models.TextField(_("about me"), max_length=255, blank=True, null=True)
-    photo = models.ImageField(_("photo"), blank=True, null=True)
-    location = models.TextField(_("location"), max_length=255, blank=True, null=True)
-    phone_number = PhoneNumberField(_("phone number"), blank=True, null=True)
+    last_name = models.CharField(_("last name"), max_length=255, blank=True)
```

Comment:
@kilemensi  won't we face the same duplicate issues here given some profiles won't have `last_name`, `bio`, e.t.c?

Reply:
`last_name`, `bio`, etc., don't have `unique=True` @esirK

Reply:
aah yeah. seen.